### ISSUE TITLE
Mark not maintained

### ISSUE THREAD

**Issue by watermark** [components/foundation#28]
The repo is years out of date.  It would be nice to mark the repo as not maintained in the readme.

**Comment by ericlemerdy** [comment]
I agree. Especially since [foundation-sites](https://www.npmjs.com/package/foundation-sites) is now on npm sine version 6.x.
